ncj/3dsmax/scripts/PrepareVrayServices.bat

```diff
@@ -0,0 +1,19 @@
+@echo off
+IF %4=="VRayAdv" (
```

Comment:
Would be good to assign variables at the top for clarity.

set VRAY_RENDERER=%4

Reply:
Or use powershell